roles/glances/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+#########################################################################
+# Title:         Cloudbox Community: Glances Role                       #
+# Author(s):     Satz                                                   #
+# URL:           https://github.com/Cloudbox/Community                  #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Set DNS Record on CloudFlare"
+  include_role:
+    name: cloudflare
+  vars:
+    subdomain: glances
+  when: cloudflare_enabled
+
+- name: Stop and remove any existing container
+  docker_container:
+    name: glances
+    state: absent
+
+- name: Create glances directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }} recurse=yes"
+  with_items:
+      - /opt/glances/
+      - /opt/glances/config
+
+- name: "Check if glances.conf file exists"
+  stat:
+    path: "/opt/glances/config/glances.conf"
+  register: conf_env
+
+- name: "Copy conf file if it doesnt exist"
+  copy:
+    src: ./../config/glances.conf
```

Comment:
Files get placed in `/role/files` folder.

See https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#directory-layout

Reply:
Fixed as per the directory layout. 
thanks for pointing out.